tensorflow_model_optimization/python/examples/clustering/keras/imdb/BUILD

```diff
@@ -50,3 +50,32 @@ py_binary(
         "//tensorflow_model_optimization/python/core/clustering/keras:cluster_config",
     ],
 )
+
+py_strict_library(
+    name = "imdb_utils",
+    srcs = [
+        "imdb_utils.py",
+    ],
+    srcs_version = "PY3",
+    deps = [
+        # numpy dep1,
+        # tensorflow dep1,
+        "//tensorflow_model_optimization/python/core/clustering/keras:cluster",
+        "//tensorflow_model_optimization/python/core/clustering/keras:cluster_config",
+    ],
+)
+
+py_binary(
+    name = "imdb_multiple_cells",
+    srcs = [
+        "imdb_multiple_cells.py",
+    ],
+    python_version = "PY3",
+    deps = [
+        # numpy dep1,
```

Comment:
what do these comments refer to?

Reply:
They are not essential so I deleted them in all places. Thanks.